docs/book/helpers/partial.md

```diff
@@ -4,6 +4,11 @@ The `Partial` view helper is used to render a specified template within its own
 variable scope. The primary use is for reusable template fragments with which
 you do not need to worry about variable name clashes.
 
+> Note that it is also possible to render a template within its own variable
+> scope by calling `RendererInterface::render()` with a `$values` argument.
+> See the documentation for [`PhpRenderer`](../php-renderer.md#render) for
+> more information.
+
```

Comment:
This note could be an interesting information for the user but it should be moved to the end and not added in the introduction of this helper. And it should also be highlighted that there is a difference in the behaviour of the method `render` and the partial helper.
Please add also a headline to the blockquote; third order (`###`).